mlrun/datastore/targets.py

```diff
@@ -1516,6 +1516,23 @@ def as_df(self, columns=None, df_module=None, **kwargs):
 
 
 class KafkaTarget(BaseStoreTarget):
+    """
+    Kafka target storage driver, used to write data into kafka topics.
+    example::
+        # define target
+        kafka_target = KafkaTarget(
+            name="kafka", path="my_topic", brokers="localhost:9092"
+        )
+        # ingest
+        stocks_set.ingest(stocks, [kafka_target])
+    :param name:                target name.
+    :param path:                topic name e.g. "my_topic"
+    :param bootstrap_servers:   Deprecated. Use the brokers parameter instead.
```

Comment:
Since it's deprecated, to ease readability, please move below "brokers"

Reply:
The docstring order should match the actual parameter order... Which can't be changed for bwc.